### ISSUE TITLE
Firefox test fail over sauce labs with vs 5 when calling setValue()

### ISSUE THREAD

**Issue by pmerwin** [webdriverio/webdriverio#3443]
sauce-service and wdio

**Environment (please complete the following information):**
 - **WebdriverIO version:** [5.4.3]
 - **Mode:** [testrunner]
 - **If WDIO Testrunner, running sync/async:** [sync]
 - **Node.js version:** [8.10]
 - **NPM version:** [6.5.0]
 - **Browser name and version:** [ff vs 64]
 - **Platform name and version:** [osx or win 10]
 - **Additional wdio packages used (if applicable):** ["@wdio/sauce-service": "^5.4.3"]

**Config of WebdriverIO**
An example of how you configured WebdriverIO config

```

    user: config.saucelabs.user,
    key: config.saucelabs.key,
    hostname: 'ondemand.saucelabs.com',
    maxInstances: 3,
    capabilities: [
        {
            browserName: 'firefox',
            platform: 'OS X 10.12'
            // extendedDebugging: true,
            // version: '57.0',
            // screenResolution: '1280x960',
            // 'moz:firefoxOptions': {args: ['-headless']}
        },
    ],
    mochaOpts: {
        ui: 'bdd',
        timeout: 360000,
        bail: true
    },
    services: ['sauce'],
    reporters: ['dot', 'spec'],
    coloredLogs: true,
    deprecationWarnings: false
});

```

**Describe the bug**
When setvalue() calls sendKeys we get a null pointer exception, but only on sauce vm's, all is well locally

**To Reproduce**

call setValue on a text filed:
'this.username.setValue(username)'

**Expected behavior**
Set value succeeds as it does when running locally or with vs 4.

**Log**
https://app.saucelabs.com/tests/ea1cddc8a7844305b7f728fbc523a3f1/logs#13

```
"HTTPStatus": 500,
    "result": {
      "stackTrace": [
        {
          "moduleName": null,
          "methodName": "setJsonParameters",
          "classLoaderName": "app",
          "fileName": "SendKeys.java",
          "className": "org.openqa.selenium.remote.server.handler.SendKeys",
          "moduleVersion": null,
          "nativeMethod": false,
          "lineNumber": 41,
          "hCode": -1339759797,
          "class": "java.lang.StackTraceElement"
        },
        {
          "moduleName": null,
          "methodName": "handle",
          "classLoaderName": "app",
          "fileName": "ResultConfig.java",
          "className": "org.openqa.selenium.remote.server.rest.ResultConfig",
          "moduleVersion": null,
          "nativeMethod": false,
          "lineNumber": 99,
          "hCode": 1308094107,
          "class": "java.lang.StackTraceElement"
        },
```

**Additional context**

> Please use the correct markdown

**Comment by pmerwin** [comment]
We think this is because sauce is using a very old version of selenium, looking for a way to make that a newer version now

**Comment by pmerwin** [comment]
happens with newer selnium too:

12:58:12.579 INFO - Selenium build info: version: '3.9.1', revision: '63f7b50'
12:58:12.581 INFO - Launching a standalone Selenium Server on port 4443
2019-01-28 12:58:13.048:INFO::main: Logging initialized @1257ms to org.seleniumhq.jetty9.util.log.StdErrLog
2019-01-28 12:58:13.236:INFO:osjs.Server:main: jetty-9.4.7.v20170914, build timestamp: 2017-11-21T13:27:37-08:00, git hash: 82b8fb23f757335bb3329d540ce37a2a2615f0a8
2019-01-28 12:58:13.276:WARN:osjs.SecurityHandler:main: ServletContext@o.s.j.s.ServletContextHandler@18d87d80{/,null,STARTING} has uncovered http methods for path: /
2019-01-28 12:58:13.285:INFO:osjsh.ContextHandler:main: Started o.s.j.s.ServletContextHandler@18d87d80{/,null,AVAILABLE}
2019-01-28 12:58:13.342:INFO:osjs.AbstractConnector:main: Started ServerConnector@6e0f2aaf{HTTP/1.1,[http/1.1]}{0.0.0.0:4443}
2019-01-28 12:58:13.342:INFO:osjs.Server:main: Started @1551ms
12:58:13.343 INFO - Selenium Server is up and running on port 4443

**Comment by mgrybyk** [comment]
may it be the same issue that command is sent in W3C way but saucelabs expects jsonwp.

We can find it out if it is possible to run same test in chrome instead of firefox. If chrome passes (isW3C is false when using chrome) then it is most likely the same issue

**Comment by pmerwin** [comment]
Yep chrome is fine, and selenium version is not making a difference....

[log.json.log](https://github.com/webdriverio/webdriverio/files/2805099/log.json.log)

**Comment by mgrybyk** [comment]
I can see 
```
   "request": {
        "text": "..."
    },
```
maybe value with string array is expected

**Comment by pmerwin** [comment]
I fixed it by doing this:

```
{
            browserName: 'firefox',
            platformName: 'OS X 10.12',
            // isW3C: true
            // extendedDebugging: true,
            browserVersion: 'latest',
            'sauce:options': {
                seleniumVersion: '3.11.0',
            }
        },
```

**Comment by mgrybyk** [comment]
@pmerwin setting selenium version to 3.11.0 helped? Why this version?

**Comment by pmerwin** [comment]
Yeah sauce was loading 3.40 of selenium server, I found a way to make it load the 3.11 and that fixed the issue
